### PR TITLE
Fixed passing labels to model in adapter training notebook

### PR DESCRIPTION
**Pull request by calpt**
Quick fix for the adapter training notebook by explicitly avoiding the label columns to be removed by the HF Trainer. Diff is pretty ugly, the only relevant changes is the last arg in:

```python
training_args = TrainingArguments(
    learning_rate=1e-4,
    num_train_epochs=6,
    per_device_train_batch_size=32,
    per_device_eval_batch_size=32,
    logging_steps=200,
    output_dir="./training_output",
    overwrite_output_dir=True,
    # The next line is important to ensure the dataset labels are properly passed to the model
    remove_unused_columns=False,
)
```

### CODE REVIEW

**Review by hSterz — COMMENTED** [review]
The training seems to work but when I execute this the mapping from id to label is not working correctly. The classifier outputs:
'label': 'LABEL_1'
but as I understand it the model should instead output 👍

**Comment by calpt** [comment]
Not sure if I understood you correctly. What should the classifier output instead of `LABEL_1`?

**Comment by calpt** [comment]
Thanks, great catch! Fixed it.

**Review by hSterz — APPROVED** [review]
Now it looks good
